### PR TITLE
Helm3 flag

### PR DESCRIPTION
**Pull request by brendarearden**
**Related PR**
https://github.com/rancher/rancher/pull/25300

**Note: ** Do not merge until rancher PR ready to merge

### CODE REVIEW

File: apis/management.cattle.io/v3/catalog_types.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ type CatalogSpec struct {
 	CatalogKind string `json:"catalogKind,omitempty"`
 	Username    string `json:"username,omitempty"`
 	Password    string `json:"password,omitempty" norman:"type=password"`
+	HelmVersion string `json:"helmVersion,omitempty" norman:"noupdate"`
```
Comment (luthermonson) [inline apis/management.cattle.io/v3/catalog_types.go]: shouldnt this be in status too?

Comment (brendarearden) [reply apis/management.cattle.io/v3/catalog_types.go]: So status are things that are not set by the user, in this case we want the user to be able to set the helmversion at the catalog level on create, so it needs to live in the spec

Comment (luthermonson) [reply apis/management.cattle.io/v3/catalog_types.go]: roger

File: apis/project.cattle.io/v3/app_types.go [hunk 2]
```diff
@@ -58,7 +58,7 @@ type AppStatus struct {
 	Notes                string            `json:"notes,omitempty"`
 	Conditions           []AppCondition    `json:"conditions,omitempty"`
 	LastAppliedTemplates string            `json:"lastAppliedTemplate,omitempty"`
-	HelmVersion          string            `json:"helmVersion,omitempty" norman:"nocreate,noupdate"`
```
Comment (dramich) [inline apis/project.cattle.io/v3/app_types.go]: Same thing?

Comment (brendarearden) [reply apis/project.cattle.io/v3/app_types.go]: same thing, just wanted it to be in the same order as the other ones I added 🤷‍♂

**Pull request merged**